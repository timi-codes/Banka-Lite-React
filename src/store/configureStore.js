import { createStore, applyMiddleware, compose } from 'redux';
import thunkMiddleware from 'redux-thunk';
import rootReducer from '@reducers';

const storeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const middlewares = [thunkMiddleware];

const store = createStore(
  rootReducer,
  {},
  storeEnhancers(applyMiddleware(...middlewares)),
);

export default store;
