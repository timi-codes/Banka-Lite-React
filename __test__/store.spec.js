import store from '../src/store/configureStore';

describe('Test REDUX STORE', () => {
  it('test redux store', () => {
    expect(store).toMatchSnapshot();
  });
});
