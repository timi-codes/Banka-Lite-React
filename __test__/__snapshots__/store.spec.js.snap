// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test REDUX STORE test redux store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
